### DISCUSSION TITLE
The type CANCoder has been deprecated since version 2024 and marked for removal

### DISCUSSION THREAD

**Discussion by kglasnapp** [wpilibsuite/2024Beta#44] [Q&A]
How do I use CANCoder in Phoenix 5? 
I tried import com.ctre.phoenix.motorcontrol.RemoteSensorSource;  but  private CANCoder absEnc; gets a warning.

**Comment by ThadHouse** [comment]
Deprecation just means marked for removal in the future. You can still use it this year if you don't want to update everything else to phoenix 6.

**Reply by kglasnapp** [reply]
I tried Phoenix 6 and was unable to get SRX Talons to work.  Cory from CTRE said I need to use Phoenix 5.

**Reply by ghost** [reply]
@kglasnapp you can use both 5 and 6 in the same project.
